boards/silabs/dev_kits/sltb010a/Kconfig.defconfig

```diff
@@ -30,3 +31,7 @@ endif # BT
 
 config REGULATOR
 	default y if SI7210
+
+config BOARD_LATE_INIT_HOOK
+	def_bool y if UART_GECKO
```

Comment:
I believe `board_late_init_hook()` need to be called unconditionally. Not only if UARt is enabled.